source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -128,17 +128,116 @@ the Server manual documentation.
    from the Query API expression provided in each example. However,
    both expressions will produce the same aggregation result.
 
+Array Operations
+~~~~~~~~~~~~~~~~
+
+You can perform an array operation on a value of type ``MqlArray``
+using the methods described in this section.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Java Method
+     - Aggregation Pipeline Operator
+
+   * - `concat() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#concat(com.mongodb.client.model.mql.MqlArray)>`__
+     - :manual:`$concatArrays </reference/operator/aggregation/concatArrays/>`
+
+   * - `contains() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#contains(T)>`__
+     - :manual:`$in </reference/operator/aggregation/in/>`
+
+   * - `elementAt() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#elementAt(int)>`__
+     - :manual:`$arrayElemAt </reference/operator/aggregation/arrayElemAt/>`
+
+   * - `filter() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#filter(java.util.function.Function)>`__
+     - :manual:`$filter </reference/operator/aggregation/filter/>`
+
+   * - `first() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#first()>`__
+     - :manual:`$first </reference/operator/aggregation/first/>`
+
+   * - `last() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#last()>`__
+     - :manual:`$last </reference/operator/aggregation/last/>`
+
+   * - `map() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#map(java.util.function.Function)>`__
+     - :manual:`$map </reference/operator/aggregation/map/>`
+
+   * - `size() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#size()>`__
+     - :manual:`$size </reference/operator/aggregation/size/>`
+
+   * - `slice() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#slice(int,int)>`__
+     - :manual:`$slice </reference/operator/aggregation/slice/>`
+
+.. include:: /includes/fundamentals/agg-mql-diff.rst
+
+Suppose you have documents describing upcoming events that you might want
+to sign up for. Each event document contains an array of nested
+documents corresponding to each date that the event is hosted.
+Each nested document also contains the number of available seats and
+current number of sign-ups for that date's event. You want to only see
+events that still have open seats.
+
+To find the number of open seats for an event, subtract the ``signups``
+field from the ``seats`` field by using the ``subtract()`` method. To
+filter only for events with open seats, pass a predicate to the
+``filter()`` method that checks if the difference between ``seats`` and
+``signups`` is greater than ``0`` by using the ``gt()`` method. To
```

Comment:
[suggestion]

I think this is repetitive with the previous sentence since you already defined how you find the number of open seats. Recommend changing this to tighten the language.

```suggestion
``filter()`` method that checks if the number of open seats
is greater than ``0`` by using the ``gt()`` method. To
```